docs/README.md

```diff
@@ -37,14 +37,19 @@ installed. Also install the following libraries:
 
 ```sh
 $ sudo gem install jekyll jekyll-redirect-from rouge
-# Following is needed only for generating API docs
-$ sudo pip install sphinx pypandoc mkdocs numpy
-$ sudo Rscript -e 'install.packages(c("knitr", "devtools", "testthat", "rmarkdown"), repos="https://cloud.r-project.org/")'
-$ sudo Rscript -e 'devtools::install_version("roxygen2", version = "5.0.1", repos="https://cloud.r-project.org/")'
 ```
 
 Note: If you are on a system with both Ruby 1.9 and Ruby 2.0 you may need to replace gem with gem2.0.
 
+If you'd like to generate API docs, you'll need to [install Pandoc](https://pandoc.org/installing.html) and also
```

Comment:
I would say `... you'll need to [install Pandoc](https://pandoc.org/installing.html) for R documentation and also ...`